packages/aws-cdk-lib/assertions/lib/template.ts

```diff
@@ -54,7 +54,7 @@ export class Template {
 
   private constructor(template: { [key: string]: any }, templateParsingOptions: TemplateParsingOptions = {}) {
     this.template = template as TemplateType;
-    if (!templateParsingOptions?.skipCyclicalDependenciesCheck ?? true) {
+    if (!templateParsingOptions.skipCyclicalDependenciesCheck) {
```

Comment:
Leaving a comment to confirm that `skipCyclicalDependenciesCheck` is meant to default to `false`, as is the behavior here.